pkgs/timeable/timeable.go

```diff
@@ -0,0 +1,11 @@
+package timeable
+
+import "time"
+
+// A Timeable is an entity that holds a create and update time.
+type Timeable interface {
```

Comment:
no need for a dedicated package here. this should be in crudcore

Reply:
that means ill have to import `underwater` pkg for this specific case, is that fine ?